extension/httpforwarderextension/metadata.yaml

```
@@ -0,0 +1,9 @@
+type: http_forwarder
+
+status:
+  class: extension
+  stability:
+    beta: [extension]
+  distributions: [contrib, splunk, sumo]
```

This causes the distribution reports to include the `http_forwarder` extension twice. How about we make this list empty and only update when the distros actually switch to the new component?
```suggestion
  distributions: []
```

---

Got it. I will also deprecate the old extension.